source/_patterns/01-molecules/components/social-links.mustache

```diff
@@ -1,3 +1,4 @@
+<!-- No aria role contentinfo needed in Social Links as this is always placed within the footer -->
```

Comment:
Use Mustache comments (`{{! ... }}`), as we don't actually want this appearing in the real HTML.